packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts

```diff
@@ -373,7 +373,8 @@ export class AwsSqs implements INodeType {
 
 			let responseData;
 			try {
-				responseData = await awsApiRequestSOAP.call(this, 'sqs', 'GET', `${queuePath}/?Action=${operation}&` + params.join('&'));
+				const properCasedOperation = operation[0].toUpperCase() + operation.slice(1);
+				responseData = await awsApiRequestSOAP.call(this, 'sqs', 'GET', `${queuePath}/?Version=2012-11-05&Action=${properCasedOperation}&` + params.join('&'));
```

Comment:
we should apply this version to loading queues as well https://github.com/n8n-io/n8n/blob/48690d17c13edf49e0f32e28e8a51673e00118ed/packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts#L272